tests/publiccloud/download_repos.pm

```diff
@@ -74,6 +77,8 @@ sub run {
             script_run("echo 'Downloading $maintrepo ...' >> ~/repos/qem_download_status.txt");
             my ($parent) = $maintrepo =~ 'https?://(.*)$';
             my ($domain) = $parent =~ '^([a-zA-Z.]*)';
+            record_info("test", "$parent");
```

Comment:
just not to forget later I will a reminder here that this needs to be removed before we will commit